src/content/3.12/Enriched Categories.tex

```diff
@@ -165,20 +183,48 @@ \section{Enriched Category}
 
 \begin{figure}[H]
 \centering
-\includegraphics[width=80mm]{images/compcoherence.jpg}
+\begin{tikzcd}[column sep=large]
+(\cat{C}(c,d) \otimes \cat{C}(b,c)) \otimes \cat{C}(a,b)
+\arrow[r, "\circ\otimes\id"]
+\arrow[dd, "\alpha"]
+  & \cat{C}(b,d) \otimes \cat{C}(a,b)
+    \arrow[dr, "\circ"] \\
+  & & \cat{C}(a,d) \\
+\cat{C}(c,d) \otimes (\cat{C}(b,c) \otimes \cat{C}(a,b))
+\arrow[r, "\id\otimes\circ"]
+  & \cat{C}(c,d) \otimes \cat{C}(a,c)
+    \arrow[ur, "\circ"]
+\end{tikzcd}
 \end{figure}
 
 \noindent
 Unit laws are likewise expressed in terms of unitors:
 
 \begin{figure}[H]
-\centering
-\includegraphics[width=70mm]{images/rightid.jpg}
-\end{figure}
-
-\begin{figure}[H]
-\centering
-\includegraphics[width=70mm]{images/leftid.jpg}
+  \centering
+  \begin{subfigure}
+    \centering
+    \begin{tikzcd}[row sep=large]
+      \cat{C}(a,b) \otimes i
+      \arrow[rr, "\id \otimes j_a"]
+      \arrow[dr, "\rho"]
+        & & \cat{C}(a,b) \otimes \cat{C}(a,a)
+            \arrow[dl, "\circ"] \\
+        & \cat{C}(a,b)
+    \end{tikzcd}
+  \end{subfigure}
+  \hspace{1cm}
+  \begin{subfigure}
+    \centering
+    \begin{tikzcd}[row sep=large]
+      i \otimes \cat{C}(a,b)
+      \arrow[rr, "j_b \otimes \id"]
+      \arrow[dr, "\rho"]
```

Comment:
I think this should be a `\lambda`:
![image](https://user-images.githubusercontent.com/601206/34457338-9e509c5a-edb6-11e7-95ef-c613ee6ed001.png)